### ISSUE TITLE
The BG color is not being set to the whole page

### ISSUE THREAD

**Issue by Brunomachadob** [baires/shouldideploy#499]
I saw some refactoring of CSS classes here on the latest commits, maybe one of them is the cause of this? Or it could also be a change on the Chrome engine.

I'm using:
```
Google Chrome
Version 90.0.4430.212 (Official Build) (x86_64)
```

Actually also happening on Safari

```
Safari
Version 14.0.2 (16610.3.7.1.9)
```

![image](https://user-images.githubusercontent.com/4688763/120007882-b081bb80-bfda-11eb-935a-e30bd75d3ed0.png)

**Comment by Brunomachadob** [comment]
I'm not a CSS expert, but apparently the error comes from the `<div id="__next"></div>` that is not expanding to the whole page.

From the Chrome inspector I could solve it by using `width: 100%` on it.

And changing the `.wrapper` to 
```diff
.wrapper {
-justify-content: space-between;
+justify-content: center;
}
```

But it would be nice to get an expert advice on how to better fix this, maybe @HarshaVardhanNakkina that recently improved it?

**Comment by HarshaVardhanNakkina** [comment]
thank you so much for pointing it out. I just opened a pull request for the fix https://github.com/baires/shouldideploy/pull/500, it is deployed to preview, seems good so far. Please verify it
![image](https://user-images.githubusercontent.com/48801182/120019011-8bef0900-c005-11eb-8afc-a566f4958470.png)

**Comment by baires** [comment]
solved on #500
